docs/en/03_How_tos/adding_a_calendar.md

```diff
@@ -30,36 +31,41 @@ parameters can then be passed to `AllEvents` to produce the single-month listing
 We can create a function that will parse the parameters and provide us with a neat array of SQL-safe variables. We start
 by getting the parameters from the `SS_HTTPRequest`.
 
-	:::php
-	public function parseEventParams() {
-		$year = $this->request->getVar('year');
-		$month = $this->request->getVar('month');
+```php
+public function parseEventParams() 
+{
+    $year = $this->request->getVar('year');
+    $month = $this->request->getVar('month');
+```
 
 We need to sanitise them before doing anything else. We could use `Convert` helper class to do it, or simply cast the
 value:
 
-	:::php
-		if (isset($year)) $year = (int)$year;
-		if (isset($month)) $month = (int)$month;
+```php
+    if (isset($year)) $year = (int)$year;
+    if (isset($month)) $month = (int)$month;
+```
 
 Next, since the month without year doesn't have much sense, we make sure that we check for validity and reset to
 defaults if needed. This assures us that we don't end up displaying a listing of all site events.
 
-	:::php
-		// Default to current month if either not provided.
-		if (!isset($month) || !isset($year)) {
-			$year = SS_Datetime::now()->Format('Y');
-			$month = SS_Datetime::now()->Format('m');
-		}
+```php
+    // Default to current month if either not provided.
+    if (!isset($month) || !isset($year)) {
+        $year = SS_Datetime::now()->Format('Y');
```

Comment:
```diff
- $year = SS_Datetime::now()->Format('Y');
+ use SilverStripe\ORM\FieldType\DBDatetime;
+ // ...
+ $year = DBDatetime::now()->Format('y');

- $month = SS_Datetime::now()->Format('m');
+ $month = DBDatetime::now()->Format('MM');
```

Note the different Format arguments (PHP date format switched to CLDR date format)